### ISSUE TITLE
Gravel ores

### ISSUE THREAD

**Issue by buttilda** [buttilda/Another-One-Bites-the-Dust#115]
similar to the ones added by TiCon but with the dynamism of AOBD

**Comment by Voxel-Friend** [comment]
People can harvest gravel ores by causing them to fall on a torch or flower. If you can prevent this I'd say it's a great idea. Great implementation of shovel mining levels.

**Comment by UndeadZeratul** [comment]
Between the shovel mining levels for gravel ores, and the axe mining levels of the oreberry bushes, it's about time mods utilize them.

I do agree though, if you can prevent the ore dropping when falling on something like a torch, or pressure plate, or whatever else people in vanilla use to harvest massive amounts of sand/gravel, that would be great; might even re-add them to my worldgen.

**Comment by buttilda** [comment]
Could simply make them not fall with gravity and work like dirt?

**Comment by UndeadZeratul** [comment]
Let me guess, the falling is hardcoded to break itself when it collides with such a block?

**Comment by buttilda** [comment]
It drops itself as an item if it falls into the block space of a block it can't replace.
e.g.: it can replace air or tall grass, so it will just turn "solid". It can't replace a torch, though it can enter its space due to the torch not having a collision box, so it drops itself as an item (otherwise you'd either lose the torch or the gravel, which is not ideal)

**Comment by UndeadZeratul** [comment]
Losing the torch would be ok in my opinion, it's just a torch.  Is it possible to make it drop the block it falls into as an item?

Either way, preventing the physics would be the simplest answer, but then it makes them inconsistent with their TiC counterparts.

**Comment by hron84** [comment]
@UndeadZeratul It's not about a torch. Falling blocks drops itself on slabs to and you can imagine a diamond slab what you definitelly not want to loose just because a falling block. But anyway, this is not how Minecraft works. Falling blocks are falling and it already has a well-defined behavior that shouldn't changed.

I do not like to not make these ores falling. If they're rare enough, it does not causes a big issue if someone mine them with a torch. TiCon gravel ores works on this way and "Gravel ores" should have a consistent behaviour.

**Comment by buttilda** [comment]
I agree with @hron84. It's a small exploit that doesn't really break progression in any way and just makes harvesting ores take a tiny bit less time. You'd still need a shovel that can mine the ore in order to mine the bottom one. And the gravel ore clusters are small and short, you don't get more than 4-5 in a pile, which means you won't even save that much time by using the torch-mining technique.
